examples/basic/README.md

```diff
@@ -1,9 +1,18 @@
 # Basic Splunk Otel Distro example
 
+
+## Pre-Requisite for Ruby versions < 3.0
+There was a change to the C++ version used to build the Thrift Ruby extension, a transitive dependency of the `opentelemetry-exporter-jaeger` gem, from building properly.  Running `./console.rb` will spit out a lengthy error message that includes:
+```bash
+binary_protocol_accelerated.c:404:68: error: '(' and '{' tokens introducing statement expression appear in different macro expansion contexts
+```
+
+In order to remediate, you must manually install this gem into your Ruby `gems` path by executing `gem install thrift -- --with-cppflags="-Wno-compound-token-split-by-macro"`.  
```

Comment:
We need to specify the version of the `thrift` gem here and explain how to find it, because if you install a version which doesn't match requirements specified by `opentelemetry-exporter-jaeger` then it won't work.